src/proto/grpc/extension/extension.proto

```diff
@@ -0,0 +1,69 @@
+// Copyright 2018 gRPC authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+syntax = "proto3";
+
+package grpc.extension;
+
+message ExtensionConfig {
+  repeated ClientChannelConfig client_channel = 1;
+}
+
+message ClientChannelConfig {
+  // A list of connection targets which use this channel config.
+  repeated string target = 1;
+
+  // The pooling config for the underlying channel pool.
```

Comment:
This description doesn't say much ... instead, you should describe what channel pools are, and their purposes.